src/Tasks/Microsoft.NET.Build.Tasks/SingleProjectInfo.cs

```diff
@@ -70,11 +78,29 @@ public static SingleProjectInfo Create(string projectPath, string name, string f
                 string version = null; // it isn't possible to know the version from the MSBuild info.
                                        // The version will be retrieved from the project assets file.
 
-                List<ResourceAssemblyInfo> resourceAssemblies = new List<ResourceAssemblyInfo>();
-
                 projectReferences.Add(
                     sourceProjectFile,
-                    new SingleProjectInfo(sourceProjectFile, name, version, outputName, resourceAssemblies));
+                    new SingleProjectInfo(sourceProjectFile, name, version, outputName, null, null));
```

Comment:
nit: use named arguments for null literals.

Reply:
Fixed.